cmake/OpenCVFindLibsGUI.cmake

```diff
@@ -47,6 +47,9 @@ if(WITH_QT)
       find_package(Qt${QT_VERSION_MAJOR} COMPONENTS OpenGL QUIET)
       if(Qt${QT_VERSION_MAJOR}OpenGL_FOUND)
         set(QT_QTOPENGL_FOUND ON)  # HAVE_QT_OPENGL is defined below
+        if(QT_VERSION_MAJOR GREATER 5) # QGL -> QOpenGL
+          find_package(Qt${QT_VERSION_MAJOR} COMPONENTS OpenGLWidgets QUIET)
```

Comment:
We should properly check find_package() result.

IMHO, it should be merged with the check above.

Reply:
I agree, a check here would be appropriate.